setup.py

```diff
@@ -49,11 +49,14 @@
             "tox>=3.5.3,<4.0",
             # linting
             "flake8>=3.0,<4.0",
-            "isort>=4.3,<5.0",
+            'isort>=5.1.4,<6.0;python_version>="3.6"',
```

Comment:
If we're going to make the switch to isort 5 we should be able to trim our config down to:

```
[isort]
profile = black
known_first_party = tests, globus_sdk
```

Reply:
I didn't know about `profile = black`. Thank goodness.
`flake8` rejected the idea of such a mode.

Let's save that for follow-up, if that's okay with you.